codegen/projections/rails_json/Steepfile

```diff
@@ -3,8 +3,8 @@
 D = Steep::Diagnostic
 
 target :app do
-  signature 'sig'
   library 'hearth'
+  signature 'sig'
```

Comment:
As I was working on various fixes I was trying to find a good set of type checks - I think we may want to do something like what we do in v3 using rbs:test. I think that gives the closest to validating what we care about - validation of the end user public types. 

I did run into issues with double that I can't figure out how to solve without just not using doubles (ie creating a client and mocking vs passing a double).

Reply:
Update for posterity: move to a v3 like rbs:test in CI.